com.airtame.Client.appdata.xml

```diff
@@ -26,6 +26,7 @@
   </categories>
   <!-- https://help.airtame.com/hc/en-us/articles/5348392729117-Changelog-Product-Changes-Airtame-Application- -->
   <releases>
+    <release version="4.5.3" date="2022-11-15" />
```

Comment:
My best guess is that it was released on `2022-08-12`. https://snapcraft.io/airtame-application was last updated on that date and the last-modified of `https://downloads.airtame.com/app/latest/linux/Airtame-4.5.3.snap` is also that date.

Perhaps you could add the changelog for 4.5.0 and 4.5.2 as documented [here](https://help.airtame.com/hc/en-us/articles/5348392729117-Changelog-Product-Changes-Airtame-Application-)?

Reply:
4.5.2 is on 2022-09-12 per the changelog, I don't think 4.5.3 is 2022-08-12? Anyways I added the dates.